plugin/plugins/wechat_integration/static/index.html

```
@@ -0,0 +1,94 @@
+
+        <!-- QR Code Card -->
+        <section class="qrcode-section qrcode-card" id="qrcode-card">
+            <div class="status-head">
+                <span class="status-title" data-i18n="ui.qrcode.title">登录二维码</span>
+                <span id="login-status-pill" class="status-pill" data-i18n="ui.status.idle">未登录</span>
+            </div>
+            <div class="qrcode-body">
+                <!-- Hint text (hidden when logged in or QR active) -->
+                <p class="field-hint" id="qrcode-hint" data-i18n="ui.qrcode.hint">请使用手机微信扫一扫，在手机上确认登录。二维码有效期约 5 分钟。</p>
+                <div class="qrcode-wrapper">
+                    <img id="qrcode-image" alt="QR Code" src="" style="display:none;" />
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)